From f1e7194917611231467dfc90306db380ba5c577f Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Wed, 22 Oct 2014 12:30:15 -0400
Subject: [x86] build: Add arch/x86/purgatory/ make generated files to gitignore

Message-id: <1413981015-15686-1-git-send-email-prarit@redhat.com>
Patchwork-id: 98043
O-Subject: [RHEL7.1 PATCH BZ 1155577] x86/build: Add arch/x86/purgatory/ make generated files to gitignore
Bugzilla: 1155577
RH-Acked-by: Vivek Goyal <vgoyal@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1155577

commit 4ea48a01bb1a99f4185b77cd90cf962730336cc4
Author: Shuah Khan <shuahkh@osg.samsung.com>
Date:   Mon Sep 29 12:41:56 2014 -0600

    x86/build: Add arch/x86/purgatory/ make generated files to gitignore

    The following generated files are missing from gitignore
    and show up in git status after x86_64 build. Add them
    to gitignore.

        arch/x86/purgatory/kexec-purgatory.c
        arch/x86/purgatory/purgatory.ro

    Signed-off-by: Shuah Khan <shuahkh@osg.samsung.com>
    Link: http://lkml.kernel.org/r/1412016116-7213-1-git-send-email-shuahkh@osg.samsung.com
    Signed-off-by: Ingo Molnar <mingo@kernel.org>

Cc: vgoyal@redhat.com
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/.gitignore b/arch/x86/.gitignore
index 7cab8c0..aff152c 100644
--- a/arch/x86/.gitignore
+++ b/arch/x86/.gitignore
@@ -1,4 +1,6 @@
 boot/compressed/vmlinux
 tools/test_get_len
 tools/insn_sanity
+purgatory/kexec-purgatory.c
+purgatory/purgatory.ro
 
-- 
1.7.1